.circleci/config.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           command: cargo generate-lockfile
       - restore_cache:
           keys:
-            - v4-cargo-cache-{{ arch }}-{{ checksum "Cargo.lock" }}
+            - v5-cargo-cache-{{ arch }}-{{ checksum "Cargo.lock" }}
```

Comment:
Is manually bumping the `v5` a cache-busting thing local to this CI or is this something tied into the Rust toolchain where `1.51.0` always runs `v5`?